tests/engines/registries/integration/components/registries-navbar/component-test.ts

```diff
@@ -93,7 +97,7 @@ module('Registries | Integration | Component | registries-navbar', hooks => {
         assert.dom('a[data-test-join]').hasText(`${t('navbar.join')}`);
         assert.dom('a[data-test-join]').hasAttribute(
             'href',
-            'http://example.com?campaign=osf-registries&next=http%3A%2F%2Flocalhost%3A5000%2FFakeURL',
+            `http://example.com?campaign=osf-registries&next=${osfUrlEncoded}FakeURL`,
```

Comment:
I changed my osf url in local.js to localhost:4200 so menus will work, and this test broke because it was wrong.